curriculum/challenges/english/03-front-end-libraries/sass/apply-a-style-until-a-condition-is-met-with-while.english.md

```diff
@@ -25,31 +25,21 @@ There should be 10 different classes from <code>text-1</code> to <code>text-10</
 ```yml
 tests:
   - text: Your code should use the <code>@while</code> directive.
-    testString: assert(code.match(/@while /g), 'Your code should use the <code>@while</code> directive.');
+    testString: assert(code.match(/@while /g));
   - text: Your code should set an index variable to 1 to start.
```

Comment:
```suggestion
  - text: Your code should start at an index of 1.
```
I think this sounds better to me. If you feel this is out of the scope, feel free to dismiss the review

Reply:
I am ok with the verbiage change, but I can not commit your suggestion, because you deleted a space before `- text:`

Reply:
My bad, fixed it